generator/internal/scaffold_generator.cc

```diff
@@ -628,9 +627,7 @@ which should give you a taste of the $title$ C++ client library API.
   the program's console.
 
 - `GOOGLE_CLOUD_CPP_ENABLE_TRACING=rpc-streams` turns on tracing for streaming
-  gRPC calls, such as `Client::Read()`, `Client::ExecuteQuery()`, and
-  `Client::ProfileQuery()`. This can produce a lot of output, so use with
-  caution!
+  gRPC calls. This can produce a lot of output, so use with caution!
```

Comment:
It's annoying, but I think you need to make this change in the following files:
```sh
# grep -rl "gRPC calls, such as" --exclude "spanner-main.dox" | sort
accessapproval/doc/main.dox
assuredworkloads/doc/main.dox
billing/doc/main.dox
kms/doc/main.dox
logging/doc/main.dox
pubsublite/doc/main.dox
scheduler/doc/main.dox
secretmanager/doc/main.dox
tasks/doc/main.dox
webrisk/doc/main.dox
websecurityscanner/doc/main.dox
workflows/doc/main.dox
```

Reply:
Done.